app/scripts/models/account.js

```diff
@@ -462,12 +475,44 @@ define(function (require, exports, module) {
         });
     },
 
-    completePasswordReset: function (token, code, relier) {
+    /**
+     * Override set to only allow fields listed in ALLOWED_FIELDS
+     *
+     * @method set
+     */
+    set: _.wrap(Backbone.Model.prototype.set, function (func, attribute, value, options) {
+
+      var attributes;
+      // Handle both `"key", value` and `{key: value}` -style arguments.
+      if (_.isObject(attribute)) {
+        attributes = attribute;
+      } else {
+        (attributes = {})[attribute] = value;
```

Comment:
After the initial moment of *?!*, this took me about 5 seconds to parse. Not sure if that makes me stupid, but I'd find it easier to read broken over two lines:

```js
  attributes = {};
  attributes[attribute] = value;
```